patch.sh

```diff
@@ -3,13 +3,9 @@
 cd $(dirname $(readlink -f $0))
 
 cd squashfuse
-if [ ! -e ./ll.c.orig ]; then
-  patch -p1 --backup < ../squashfuse.patch
-  patch -p1 --backup < ../squashfuse_dlopen.patch
-fi
-if [ ! -e ./squashfuse_dlopen.c ]; then
-  cp ../squashfuse_dlopen.c .
-fi
-if [ ! -e ./squashfuse_dlopen.h ]; then
-  cp ../squashfuse_dlopen.h .
-fi
+git checkout ll.c Makefile.am fuseprivate.c fuseprivate.h hl.c ll.h ll_inode.c nonstd-daemon.c
+
+patch -p1 < ../squashfuse.patch
```

Comment:
Just checking, do we still need this?

Reply:
I don't know that. I guess we could use a more up to date version of squashfuse. Didn't they recently merge zstd compression which you wanted to use?

Reply:
Yes. I think the original reason for us patching it was that we needed offsets, which they also have merged.